h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/rabit/RabitTrackerH2O.java

```diff
@@ -173,7 +177,7 @@ public void run() {
                         }
                     }
                 } catch (IOException e) {
-                    e.printStackTrace();
+                    Log.debug("Exception in Rabit tracker.", e);
```

Comment:
I think this deserves a comment in the code why it is okay to ignore the exception. Also - perhaps a warning would be better then debug?

Reply:
good point, probably should kill the tracker and propagate somehow the exception to the parent thread and stop the modelling - not sure what would be the best way, though

Reply:
If you re-throw a wrapped exception here it will call uncaughtException handler. The handler can save the exception and your `waitFor` method can re-throw it later. Is it convoluted?